frontend/src/app/active-sessions/active-sessions.component.ts

```diff
@@ -47,8 +47,10 @@ export class ActiveSessionsComponent implements OnInit {
   }
 
   openReconnectDialog(session: Session): void {
-    this.dialog.open(ReconnectDialogComponent, {
-      data: session,
-    });
+    if (session.state == "Started" || session.state == "START_SESSION"){
+      this.dialog.open(ReconnectDialogComponent, {
+        data: session,
+      });
+    }
```

Comment:
The check on the button is sufficient. We don't need to verify the state here again.